angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome.html

```diff
@@ -0,0 +1,11 @@
+<div class="welcome-popup center">
+  <h1>Welcome to the new MyUW. Here are the basics.</h1>
+  <img src="img/myuw-welcome.png">
+	<div class="sr-only">
+		<p>The new MyUW makes it easier to access your content. You can switch between simple icons and more informative widgets using 'Icon View' and 'Widget View' toggles. Each of the icons on the home page links to your MyUW content, and widgets can display a dashboard of information on the home page. If you are having trouble finding what you're looking for, you can use the top search box to search through all of MyUW's content, and save it to your home page from there using the 'Add to Favorites' button.</p>
+	</div>
+  <div>
+    <button class="btn btn-lg btn-primary" ng-click="$dismiss()">Continue to MyUW</button>
+  </div>
+  <p>Or learn more at the <a href="#">KnowledgeBase</a></p>
```

Comment:
Missing link acknowledged in the pull request description.  Nonetheless, flagging with this comment : Learn more link needs to either disappear or link somewhere before this is merged so that the code stays shippable. :smile: